From 89a0e66edc29954ca5eba9daaac02853e020e750 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 14 Dec 2018 07:40:30 +0100
Subject: [PATCH 1/1] tests: enable DT overlay tests by default

Enable device tree overlay tests by default if unit tests are selected.

Overlays have been mainlined in the device tree compiler so there is no
reason anymore to disable the tests by default.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 test/overlay/Kconfig | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/test/overlay/Kconfig b/test/overlay/Kconfig
index 13c85428cba..19fa135f2da 100644
--- a/test/overlay/Kconfig
+++ b/test/overlay/Kconfig
@@ -1,11 +1,10 @@
 config UT_OVERLAY
 	bool "Enable Device Tree Overlays Unit Tests"
-	depends on OF_LIBFDT_OVERLAY
+	default y
 	depends on UNIT_TEST
+	select OF_LIBFDT_OVERLAY
 	help
 	  This enables the 'ut overlay' command which runs a series of unit
 	  tests on the fdt overlay code.
 	  If all is well then all tests pass although there will be a few
 	  messages printed along the way.
-	  Be warned that it requires an out-of-tree dtc compiler with patches
-	  to support the DT overlays, otherwise it will fail.
-- 
2.19.2

